pkg/platform/packet/packet.go

```diff
@@ -237,6 +252,25 @@ func (c *config) terraformSmartApply(ex *terraform.Executor, dnsProvider dns.DNS
 	return ex.Apply()
 }
 
+// terraformCreateReservations creates nodes that use a specific hardware
+// reservation ID
```

Comment:
```suggestion
// reservation ID.
```

Linter for that pending :smile: https://github.com/golangci/golangci-lint/pull/993

Reply:
haha, cool :-D

Reply:
Fixed, will push soon.